UnitedSets/Classes/Cell.Property.Callback.cs

```diff
@@ -13,7 +13,7 @@ void OnCurrentCellChanged()
         {
             _CurrentCell.Closed += delegate
             {
-                CurrentCell = null;
+                CurrentCell = null!;
```

Comment:
I think I'll fix the source generator for this might be better than just ignoring the fact that the value can be null.

Reply:
w/e you like:) When something breaks generated code often doesn't get compiled the error window is full of useless things.   Compiler output's first error is generally good but sometimes it was so many warnings tough to read.  Goal was just reduce the null warning spam.

Reply:
Just fix the generator on latest commit in master branch